src/ImageBackingStore.cpp

```diff
@@ -313,3 +313,15 @@ void ImageBackingStore::flush()
         m_fsfile.flush();
     }
 }
+
+size_t ImageBackingStore::name(char* filename, size_t len)
+{
+    if (!m_israw && !m_isrom)
```

Comment:
Could be just m_fsfile.isOpen(). The raw mapping also keeps the file open now, because access to 2352 byte sectors may require unaligned access.

Reply:
Great, I had thought these files were closed when switching between targets. If they're staying open now that'd be easier. Will change.

Reply:
This is done. I have it set up in the audio implementation as a pointer to the ImageBackingStore. I'm not well versed enough in C++ to feel confident about this approach; if you'd prefer something different let me know.